dnsapi/dns_freemyip.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env sh
+#
+#Author: Recolic Keghart <root@recolic.net>
+#
+########  Public functions #####################
+
+freemyip_token=90c55c84d72bf53f87e65a3c
```

Comment:
did you forget to remove this ?

Reply:
No. This token is just random-generated, for a demo.

Reply:
remove it